components/dashboard/src/projects/ConfigureProject.tsx

```diff
@@ -93,52 +125,121 @@ export default function () {
       return;
     }
     // (event.target as HTMLButtonElement).disabled = true;
-    setEditorError(null);
+    setEditorMessage(null);
     if (!!workspaceCreationResult) {
       setWorkspaceCreationResult(undefined);
     }
     try {
+      setPrebuildWasTriggered(true);
       await getGitpodService().server.setProjectConfiguration(project.id, gitpodYml);
       const result = await getGitpodService().server.createWorkspace({
         contextUrl: `prebuild/${project.cloneUrl}`,
         mode: CreateWorkspaceMode.ForceNew,
       });
       setWorkspaceCreationResult(result);
     } catch (error) {
-      setEditorError(<span>{String(error)}</span>);
+      setEditorMessage(<EditorMessage type="warning" heading="Could not run prebuild." message={String(error)}/>);
     }
   }
 
+  const onInstanceUpdate = (instance: WorkspaceInstance) => {
+    setPrebuildPhase(instance.status.phase);
+  }
+
   useEffect(() => { document.title = 'Configure Project — Gitpod' }, []);
 
-  return <div className="flex flex-col mt-24 mx-auto items-center">
-    <h1>Configure Project</h1>
-    <p className="text-gray-500 text-center text-base">Fully-automate your project's dev setup. <a className="gp-link" href="https://www.gitpod.io/docs/references/gitpod-yml">Learn more</a></p>
-    <div className="mt-4 w-full flex">
-      <div className="flex-1 m-8">
-        {editorError && <AlertBox className="mb-2">{editorError}</AlertBox>}
-        {!isEditorDisabled && <select className="w-full" defaultValue="" onChange={e => setGitpodYml(e.target.value)}>
-          <option value="" disabled={true}>…</option>
-          {Object.entries(TASKS).map(([ name, value ]) => <option value={value}>{name}</option>)}
-        </select>}
-        {!!dockerfile && <div className="flex justify-center border-b border-gray-200 dark:border-gray-800">
+  return <>
+    <Header title="Configuration" subtitle="View and edit project configuration." />
+    <div className="lg:px-28 px-10 mt-8 flex space-x-4">
+      <div className="w-96 h-96 rounded-xl overflow-hidden relative bg-gray-100 dark:bg-gray-700 flex flex-col">
```

Comment:
**suggestion**: @svenefftinge brought up an idea to allow users to resize the editor width, see [relevant discussion](https://gitpod.slack.com/archives/C021Q4BL8E8/p1629463581276700?thread_ts=1629461726.275900&cid=C021Q4BL8E8) (internal). This would require to have a visual indicator for users to grab and move around. For the first iteration we could go with a 50/50 split which also seems nice. What do you think @jankeromnes? 

| BEFORE | AFTER |
|-|-|
| <img width="1440" alt="Screenshot 2021-08-20 at 4 27 44 PM" src="https://user-images.githubusercontent.com/120486/130240445-1169c8e1-7782-4c4e-b8dc-93871db10bee.png"> | <img width="1440" alt="Screenshot 2021-08-20 at 4 27 38 PM" src="https://user-images.githubusercontent.com/120486/130240447-d8ab7677-2cbe-45d5-9955-f9ba6f65f238.png"> |

Reply:
Could work! I'll give this a try, thanks.